lib/src/picture_stream.dart

```diff
@@ -296,6 +296,7 @@ class OneFramePictureStreamCompleter extends PictureStreamCompleter {
       {InformationCollector informationCollector})
       : assert(picture != null) {
     picture.then<void>(setPicture, onError: (dynamic error, StackTrace stack) {
+	  setPicture(null);
```

Comment:
I don't think this is enough to distinguish between a picture that has an error and a picture that is still loading (because of an ongoing network fetch for example).

We probably need to pipe some callback through to here for errors.